tools/buildsys/src/manifest.rs

```diff
@@ -141,13 +141,21 @@ overhead for the GPT labels and partition alignment. The data partition will be
 automatically resized to fill the disk on boot, so it is usually not necessary
 to increase this value.
 
+`publish-image-size-hint-gib` is the desired size of the published image in GiB.
```

Comment:
I think I commented on it elsewere:

```suggestion
`publish-image-desired-size-gib` is the desired size of the published image in GiB.
```

Reply:
The reason I liked the "hint" phrasing is because it suggests that it might be ignored under some conditions. "desired" sounds like maybe the build should fail if the desire can't be satisfied.